Tests/SonarScanner.MSBuild.Tasks.IntegrationTests/TargetsTests/WriteProjectInfoFileTargetTests.cs

```diff
@@ -566,6 +226,58 @@ public void WriteProjectInfo_AnalysisFileList_FilesTypes_SpecifiedPlusDefaults()
 
         #region Miscellaneous tests
 
+        [TestMethod]
+        [TestCategory("ProjectInfo")]
+        public void WriteProjectInfo_IsNotTestAndNotExcluded()
```

Comment:
Added two new tests for _WriteProjectInfoFileTargetTests_ to check that the _SonarQubeTestProject_ and _SonarQubeExclude_ properties are calculated and serialized correctly.